authorizations/authorization-common/src/main/java/org/apache/gravitino/authorization/jdbc/JdbcAuthorizationPlugin.java

```diff
@@ -119,19 +119,18 @@ public Boolean onRoleCreated(Role role) throws AuthorizationPluginException {
     List<String> sqls = getCreateRoleSQL(role.name());
     boolean createdNewly = false;
     for (String sql : sqls) {
-      createdNewly = executeUpdateSQL(sql, "already exists");
+      createdNewly |= executeUpdateSQL(sql, "already exists");
     }
 
     if (!createdNewly) {
-      return true;
+      return false;
```

Comment:
Why not change this to just "return createdNewly"?

Reply:
it always returns false here